tests/library/locator-generator.spec.ts

```diff
@@ -542,6 +503,32 @@ it('parseLocator css', async () => {
   expect.soft(parseLocator('csharp', `Locator("css=.foo")`, '')).toBe(`css=.foo`);
 });
 
+it('parseLocator locator(options)', async () => {
+  expect.soft(parseLocator('javascript', `locator('.foo', { hasText: 'hello' })`, '')).toBe(`.foo >> internal:has-text="hello"i`);
+  expect.soft(parseLocator('javascript', `locator('.foo', { hasNotText: 'hello' })`, '')).toBe(`.foo >> internal:has-not-text="hello"i`);
+  expect.soft(parseLocator('javascript', `locator('.foo', { has: locator('div') })`, '')).toBe(`.foo >> internal:has="div"`);
+  expect.soft(parseLocator('javascript', `locator('.foo', { hasNot: locator('div') })`, '')).toBe(`.foo >> internal:has-not="div"`);
+  expect.soft(parseLocator('java', `locator(".foo", new Locator.LocatorOptions().setHasText("hello"))`, '')).toBe(`.foo >> internal:has-text="hello"i`);
+  expect.soft(parseLocator('java', `locator(".foo", new Locator.LocatorOptions().setHasNotText("hello"))`, '')).toBe(`.foo >> internal:has-not-text="hello"i`);
+  expect.soft(parseLocator('java', `locator(".foo", new Locator.LocatorOptions().setHas(locator("div")))`, '')).toBe(`.foo >> internal:has="div"`);
+  expect.soft(parseLocator('java', `locator(".foo", new Locator.LocatorOptions().setHasNot(locator("div")))`, '')).toBe(`.foo >> internal:has-not="div"`);
+  expect.soft(parseLocator('csharp', `Locator(".foo", new () { HasText = "hello" })`, '')).toBe(`.foo >> internal:has-text="hello"i`);
```

Comment:
```suggestion
  expect.soft(parseLocator('csharp', `Locator(".foo", new() { HasText = "hello" })`, '')).toBe(`.foo >> internal:has-text="hello"i`);
```

According to `dotnet format` this should be like that.

Reply:
Yes, it does.